### PR TITLE
fix: [DHIS2-12777] hide programs without trackedentitytype access

### PR DESCRIPTION
**Pull request by simonadomnisoru**

### CODE REVIEW

File: cypress/integration/WorkingLists/EventWorkingLists/EventWorkingListsUser/index.js [hunk 1]
```diff
@@ -17,9 +17,9 @@ Then('the default working list should be displayed', () => {
         '55 Male',
         '26 Female',
         '35 Male',
+        '12 Male',
```
Comment (jasminenguyennn) [inline cypress/integration/WorkingLists/EventWorkingLists/EventWorkingListsUser/index.js:20]: hey @simonadomnisoru , changing the order may still remain flaky tests, in newer version we did the [grouping by date ](https://github.com/dhis2/capture-app/blob/master/cypress/integration/WorkingLists/EventWorkingLists/EventWorkingListsUser/index.js#L13), can you have a look

File: cypress/integration/WorkingLists/EventWorkingLists/EventWorkingListsUser/index.js [hunk 2]
```diff
@@ -188,10 +188,10 @@ Then('the list should display data for the second page', () => {
         '61 Male',
         '9 Male',
         '15 Female',
-        '2 Male',
         '55 Female',
-        '14 Male',
+        '2 Male',
```
Comment (jasminenguyennn) [inline cypress/integration/WorkingLists/EventWorkingLists/EventWorkingListsUser/index.js:192]: same as above comment, can you have a look?

File: cypress/integration/WorkingLists/EventWorkingLists/EventWorkingListsUser/index.js (hunk 1 above)
Comment (simonadomnisoru) [reply cypress/integration/WorkingLists/EventWorkingLists/EventWorkingListsUser/index.js]: Hey, I updated the cypress scenarios just like in the newer versions. Thank you for the suggestion!

**Review by jasminenguyennn — APPROVED** [review]

**Review by geethaalwan — APPROVED** [review]
Tested successfully on 2.36.12 version

**Pull request merged**